CoreWebServer/src/au/gov/asd/tac/constellation/webserver/api/RestServiceServlet.java

```diff
@@ -101,7 +102,10 @@ private void callService(final HttpMethod httpMethod, final HttpServletRequest r
         try {
             response.setContentType(rs.getMimeType());
             response.setStatus(HttpServletResponse.SC_OK);
-            rs.callService(parameters, request.getInputStream(), response.getOutputStream());
+            ServiceResponse serviceResponse = rs.callService(parameters, request.getInputStream(), response.getOutputStream());
+            if (serviceResponse.getStatusCode()!=200){
```

Comment:
@Auriga2  should the `200` be replaced with `HttpURLConnection.HTTP_OK`?

Reply:
Yes Thanks! Fixed.